HouseRules_Essentials/README.md

```diff
@@ -1,35 +1,244 @@
+    "Rule": "CardEnergyFromRecyclingMultipliedRule",
+    "Config": 2.0
+  },
+    ```
 - **CardLimitModifiedRule**: Card limit is modified
+    - Change the size of the player's card hand from the default 10/11
+    - Config accepts Int e.g `15`  
+    ###### _Example JSON config for this rule_
+    ```json
+  {
+    "Rule": "CardLimitModifiedRule",
+    "Config": 20
+  },
+    ```
 - **CardSellValueMultipliedRule**: Card sell values are multiplied
+    - Increase card sale values in the shop. 
+    - Config accepts float e.g `2.5`  
+    ###### _Example JSON config for this rule_
+    ```json
+  {
+    "Rule": "CardSellValueMultipliedRule",
+    "Config": 2.0
+  },
+    ```
 - **EnemyAttackScaledRule**: Enemy attack damage is scaled
+    - Config accepts float e.g `0.85`  
+    ###### _Example JSON config for this rule_
+    ```json
+  {
+    "Rule": "EnemyAttackScaledRule",
+    "Config": 0.85
+  },
+    ```
 - **EnemyDoorOpeningDisabledRule**: Enemy door opening ability disabled
+    - Config accepts bool e.g `true`  
+    ###### _Example JSON config for this rule_
+    ```json
+  {
+    "Rule": "EnemyDoorOpeningDisabledRule",
+    "Config": true
+  },
+    ```
 - **EnemyHealthScaledRule**: Enemy health is scaled
+    - Config accepts float e.g `0.85`  
+    ###### _Example JSON config for this rule_
+    ```json
+  {
+    "Rule": "EnemyHealthScaledRule",
+    "Config": 0.85
+  },
+    ```
 - **EnemyRespawnDisabledRule**: Enemy respawns are disabled
+    - Config accepts bool e.g `true`  
+    ###### _Example JSON config for this rule_
+    ```json
+  {
+    "Rule": "EnemyRespawnDisabledRule",
+    "Config": true
+  },
+    ```
 - **GoldPickedUpMultipliedRule**: Gold picked up is multiplied
+    - Config accepts float e.g `1.25`  
+    ###### _Example JSON config for this rule_
+    ```json
+  {
+    "Rule": "GoldPickedUpMultipliedRule",
+    "Config": 1.25
+  },
+    ```
+
 - **LevelPropertiesModifiedRule**: Level properties are modified
+    - Allows customisation Loot, Chests and HealingFountains on a per-floor basis
+    - Config accepts Dictionary e.g. `{ "ParamName1", int, "ParamName2", int }`   
+    ###### _Example JSON config for this rule_
+    ```json
+    {
+    "Rule": "LevelPropertiesModifiedRule",
+    "Config": {
+      "BigGoldPileChance": 100,
+      "FloorOneHealingFountains": 9,
+      "FloorOneLootChests": 9,
+      "FloorTwoHealingFountains": 9,
+      "FloorTwoLootChests": 9,
+      "FloorThreeHealingFountains": 9,
+      "FloorThreeLootChests": 9
+    },
+    ```
 - **PieceConfigAdjustedRule**: Piece configuration is adjusted
   - See [PieceConfig.md](../docs/PieceConfig.md) for information about modifiable fields.
+  - Allows customization of many of the properties for each game Piece. Health, ActionPoints, Movement, MeleeDamage, etc
+  - Config accepts List of Lists e.g. `[ [ "PieceName1", "ParamName1", int ], [ "PieceName1", "ParamName1", int ], ... ]`   
+  ###### _Example JSON config for this rule_
+  ```json
```

Comment:
Not sure what about this format the GitHub linter doesn't like that it's highlighting it red, but something must be off here.  🤷🏾

Reply:
I spotted this when I created the PR and went back and carefully double-checked the file... couldn't see any issue... Checked in VSCode with Markdownlint... It complained about many violations, but nothing specifically around code blocks. I'll fix the markdown warnings tomorrow.

Reply:
It seems happier about this with the latest commit I just pushed. Was probably an issue with inconsistent indentation. It's much nicer editing markdown in VSCode than in Visual Studio.. I've got all the right plugins in VSCode to highlight my mistakes as I make them.